docs/source/branch.rst

```diff
@@ -8,125 +8,167 @@ The Branches File
-allowing us to vary the input draws from the Global Burden of Disease (GBD) used as well as the random seeds used by the
-simulation.
-
-Data Uncertainty
-^^^^^^^^^^^^^^^^
-Our simulations primarily rely on GBD results which are produced in draws to estimate uncertainty. In order to reflect
-the GBD's uncertainty surrounding their results in our simulations, we generally run simulations with a variety of
-output draws.
+primarily concerned with two kinds of uncertainy in our model -- :term:`parameter uncertainy<Parameter Uncertainty>`
+and :term:`stochastic uncertainty<Stochastic Uncertainty>`. The branch configuration can help us explore both sources
+of uncertainty by varying both the :term:`input draw<Input Draw>` of the parameter data and the
+:term:`seed<Random Seed>` of the simulation's random number generator.
+
+Parameter Uncertainty
+^^^^^^^^^^^^^^^^^^^^^
+Our simulations primarily rely on results from the Global Burden of Disease (GBD). GBD results are produced with
+:term:`uncertainy<Parameter Uncertainty>` represented as :term:`draws<Input Draw>`. Once we have a model we trust,
+we typically want to capture our uncertainty in the input data by running the simulation model for several different
+input draws.
 
 .. note::
-    A draw is a statistical term related to bootstrapping that has a specific meaning in the context of the GBD. The
-    implementation details vary, but the purpose is for
 
-    some quantity or measure of interest, a draw is a member of a full set of results such that, when taken
-    together, the set of draws describes at least some of the uncertainty surrounding the quantity as a result of the
-    modeling process, data uncertainty, etc. Generally, GBD results are produced in sets of 1000 draws.
+    A draw is a statistical term related to bootstrapping that has a specific meaning in the context of the GBD. The
+    implementation details vary, but the purpose is for some quantity or measure of interest, a draw is a member of
+    a full set of results such that, when taken together, the set of draws describes at least some of the uncertainty
+    surrounding the quantity as a result of the modeling process, data uncertainty, etc. Generally, GBD results are
+    produced in sets of 1000 draws.
 
-To do this, we can use the ``input_draw_count`` key in a branches file. This key specifies an integer that represents
-the number of different simulations that will be run in parallel, each with a different draw of GBD results drawn
-uniformly from the number available, which is generally 1,000. Below is a very simple branches file containing only a
-mapping that dictates the number of draws to run.
+To do this, we can use the ``input_draw_count`` key in a :term:`branch configuration<Branch Configuration>`.
+This key refers to an integer that represents the number of different input draws to generate simulations from.
 
 .. code-block:: yaml
```

Comment:
do these yaml blocks render in sphinx? They totally disappear from the github 'view file' version so I was a bit confused until I came back to the source here

Reply:
Yeah, they do.  A bunch of the directives and stuff will only show up after they render with sphinx